### PR TITLE
Feature/move lh dashboard to oss

### PR DESCRIPTION
**Pull request by diablouma**
move lh dashboard repo to oss repository preserving history

### CODE REVIEW

File: lh-dashboard/.vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (sauljabin) [inline lh-dashboard/.vscode/settings.json:1]: let's remove this

File: lh-dashboard/.gitmodules [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[submodule "littlehorse"]
```
Comment (sauljabin) [inline lh-dashboard/.gitmodules:1]: remove this

File: lh-dashboard/.github/workflows/main.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
+name: Build Dashboard
```
Comment (sauljabin) [inline lh-dashboard/.github/workflows/main.yml:1]: Let's pair in moving this @diablouma

File: lh-dashboard/.husky/pre-commit [hunk 4]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
```
Comment (sauljabin) [inline lh-dashboard/.husky/pre-commit:1]: we are using pre-commit and not husky, should we keep husky? guess we do not 🤔

File: lh-dashboard/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,12 @@
+FROM node:16.14-alpine
```
Comment (sauljabin) [inline lh-dashboard/Dockerfile:1]: @diablouma let's pair on moving this

**Review by sauljabin — CHANGES_REQUESTED** [review]
I would prefer do not put the prefix lh-, just let's call it dashboard, WDYT?

**Comment by sauljabin** [comment]
> I would prefer do not put the prefix lh-, just let's call it dashboard, WDYT?

done

File: lh-dashboard/.husky/pre-commit (hunk 4 above)
Comment (sauljabin) [reply lh-dashboard/.husky/pre-commit]: let's leave it there

File: lh-dashboard/.vscode/settings.json (hunk 1 above)
Comment (diablouma) [reply lh-dashboard/.vscode/settings.json]: done

File: lh-dashboard/.gitmodules (hunk 2 above)
Comment (diablouma) [reply lh-dashboard/.gitmodules]: done

File: lh-dashboard/.husky/pre-commit (hunk 4 above)
Comment (diablouma) [reply lh-dashboard/.husky/pre-commit]: tried with pre-commit before husky, however it has more complexity, even thought at some point we can double check this :) so we can have only 1 pre-commit tool

**Review by sauljabin — APPROVED** [review]
I guess we are going to do a rebase? @coltmcnealy-lh 
We can go ahead

**Comment by diablouma** [comment]
> I guess we are going to do a rebase? @coltmcnealy-lh We can go ahead

Starting to solve conflicts for the rebase :)

**Pull request merged**